Application/EdFi.Ods.Standard/Artifacts/Metadata/DatabaseViews.generated.json

```diff
@@ -954,40 +954,22 @@
     ]
   },
   {
-    "SchemaOwner": "interop",
-    "Name": "OperationalContextSupport",
+    "SchemaOwner": "auth",
+    "Name": "EducationOrganizationIdToEducationOrganizationId",
```

Comment:
It looks like we lost the OperationContextSupport view as part of these updates.  Its probably fine for now since we're not using that view actively from the API, but was this intentional?

Reply:
yes , as discussed on Friday's call , we are not using it ,so I cleaned up to reduce the maintenance overhead for not used code .